repostCheck.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+
+
+class RepostCheck:
+    cursor = None
+    sql = None
+
+    def __init__(self):
+        self.sql = sqlite3.connect('database/repost.db')

```

Comment:
Perhaps look into combining the schema of this with the existing submissions.db.  The existing DB is extremely simple so extending it should be fairly straightforward.

Reply:
I chose to make it a separate database pretty much exclusively because it made it far easier to test. I think you can just change the file name in repostCheck.py to database.db and it should work (I don't think I did a table name conflict? But if there is one just change)

If no one else does I'll do it in the evening.

Honestly merging the existing submissions table with the idStamped table should also be possible